bitnami/influxdb/Chart.yaml

```
@@ -18,23 +18,23 @@ annotations:
 apiVersion: v2
 appVersion: 2.7.6
 dependencies:
-- name: common
-  repository: oci://registry-1.docker.io/bitnamicharts
-  tags:
-  - bitnami-common
-  version: 2.x.x
+  - name: common
+    repository: oci://registry-1.docker.io/bitnamicharts
+    tags:
+      - bitnami-common
+    version: 2.x.x
 description: InfluxDB(TM) is an open source time-series database. It is a core component of the TICK (Telegraf, InfluxDB(TM), Chronograf, Kapacitor) stack.
 home: https://bitnami.com
 icon: https://bitnami.com/assets/stacks/influxdb/img/influxdb-stack-220x234.png
 keywords:
-- influxdb
-- tick
-- database
-- timeseries
+  - influxdb
+  - tick
+  - database
+  - timeseries
 maintainers:
-- name: VMware, Inc.
-  url: https://github.com/bitnami/charts
+  - name: VMware, Inc.
+    url: https://github.com/bitnami/charts
 name: influxdb
 sources:
-- https://github.com/bitnami/charts/tree/main/bitnami/influxdb
-version: 6.0.6
+  - https://github.com/bitnami/charts/tree/main/bitnami/influxdb
```

```suggestion
- https://github.com/bitnami/charts/tree/main/bitnami/influxdb
```

The YAML is still valid with both levels of indentation but it will be reverted in the next automatic update causing unnecessary commit changes.

---

Yeah, my YAML formatting plugin has been aggressively asserting itself. I'll change them back